src/main/2.0/scala/com/holdenkarau/spark/testing/receiver/InputStreamTestingContext.scala

```diff
@@ -0,0 +1,65 @@
+package com.holdenkarau.spark.testing.receiver
+
+import org.apache.spark.SparkContext
+import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.streaming.{Duration, Seconds, StreamingContext}
+
+/**
+  * Sequences the execution of user-provided code that comprises one individual test of an
+  * [[InputStreamSuiteBase]] test suite. Each individual test will typically instantiate one instance of this class.
+  *
+  * @param sparkContext        - a  [[com.holdenkarau.spark.testing.SharedSparkContext]] instance, provided by
+  *                            [[com.holdenkarau.spark.testing.StreamingSuiteBase]], which will be used to create
+  *                            an instance of  a [[org.apache.spark.streaming.StreamingContext]] that will be used
+  *                            for a run of one individual test.
+  *
+  * @param dStreamCreationFunc - a user-provided code block that uses  the StreamingContext provided by this class
```

Comment:
Hi. I fixed the comment.  Also.. in reply to your question about not using the in-memory approach... I did try with broadcast variables... but I'm somewhat new to spark streaming and failed to get that to work... so i fell back to the old school tools i know.   If you think it is important to get this done with out the admittedly clunky use of file system i can take another crack at it.      Please let me know.